halconfig/front50.yml

```diff
@@ -2,9 +2,6 @@ server:
   port: ${services.front50.port:8080}
   address: ${services.front50.host:localhost}
 
-cassandra:
-  enabled: false
-
 hystrix:
```

Comment:
@ezimanyi I was unsure whether this `hystrix` config block is necessary: it doesn't look like any of the other microservices override any hystrix settings in their config, and there is no corresponding `hystrix` block in the non-Halyard front50-web config. I left it in just to be safe, but if non-Halyard users have been fine without these Halyard-specific `hystrix` overrides then my thought was that kleat-deployed Spinnakers shouldn't need them either?

Reply:
Yeah, I suspect these are not necessary, and it's not clear how much though was put into tuning these numbers.  I'd say let's take it out and then if we do find we need this, add back in sensible values in the base config in `front50-web`.

Reply:
Sounds like a plan; I've removed it!